src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect, useRef } from "react";
+import "./Navbar.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Button, Dropdown, DropdownButton } from "react-bootstrap";
+import { List, GeoAlt } from "react-bootstrap-icons";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+import {
+  getWindowDimensions,
+  useWindowDimensions,
+} from "./../../hooks/windowSize";
+import { largeScreenNavbarData } from "./../../containers/Navbar/largeScreenNavbarData.js";
+import { smallScreenNavbarData } from "./../../containers/Navbar/smallScreenNavbarData.js";
+
+const NavbarForLargeScreens = () => {
+  return (
+    <Router>
+      <div className="navbar">
+        <div className="navbarLogo">
+          <Link to="/">LOGO</Link>
+        </div>
+        <div className="navbarLinks">
+          {largeScreenNavbarData.map((info, index) => (
+            <Link to={info.target} className={info.cName} key={index}>
+              {info.name}
+            </Link>
+          ))}
+          <DropdownButton id="dropdown-basic-button" title="Language">
+            <Dropdown.Item href="#/action-1">TUR</Dropdown.Item>
+            <Dropdown.Item href="#/action-2">ENG</Dropdown.Item>
+          </DropdownButton>
+        </div>
+        <div className="navbarProfile">
+          <Link to="/logIn">
+            <Button
+              variant="primary"
+              style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+            >
+              Log In
+            </Button>
+          </Link>
+          <Link to="/signUp">
+            <Button
+              variant="primary"
+              style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+            >
+              Sign Up
+            </Button>
+          </Link>
+        </div>
+      </div>
+    </Router>
+  );
+};
+
+const NavbarForSmallScreen = () => {
```

Comment:
can we move both the small screen and big screen navbar to seperate components and import them here ? I think that would be more clean